src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -61,6 +61,7 @@
 import java.util.Set;
 import lombok.Getter;
 import org.apache.commons.io.IOUtils;
+import org.springframework.http.HttpHeaders;
```

Comment:
Spring is far too big of a dependency to include for an HTTP header constant. Let's just define our own `USER_AGENT` constant if we'd like to use a constant (though I expect the string will only appear in exactly one place is a well-known standard, so I'd recommend foregoing the constant altogether).